tests/test_jira.py

```diff
@@ -93,9 +93,9 @@ class TestJiraIssue(unittest.TestCase):
     @patch('requests.get', side_effect=mocked_response)
```

Comment:
You want to patch `src.jira.requests.get` here, otherwise you patch isn't going to work. It's not used in this module, but in the module that runs the constructor. Let me know if you want to discuss.

Reply:
okay, that makes sense.